### ISSUE TITLE
Gmane links in PR comment don't work anymore

### ISSUE THREAD

**Issue by sschuberth** [rtyley/submitgit#34]
E.g. in https://github.com/git/git/pull/333 the "Gmane" link to http://mid.gmane.org/0102015a93d18443-c574aaf8-5656-49e8-8848-810e1fadc87a-000000@eu-west-1.amazonses.com does not work.

Maybe use https://public-inbox.org/git/ instead?

**Comment by rtyley** [comment]
Looks like the format for message-id links in is

```
https://public-inbox.org/git/0102015a93d18443-c574aaf8-5656-49e8-8848-810e1fadc87a-000000@eu-west-1.amazonses.com/
```

...which looks workable. Just need a new class in https://github.com/rtyley/submitgit/blob/95e232b59cde576d42068fc486505266f7cb01e7/app/lib/MailArchive.scala#L73-L81 to replace Gmane.

**Comment by peff** [comment]
I happened to notice this independently, and just opened #35. It's not quite there, but hopefully it gives a good start.

**Comment by rtyley** [comment]
This is fixed by #36 and 6cd943c0e - watch out for a 'public-inbox' link in the PR comment next time you submit with submitgit.
